lib/fluent/plugin/exception_detector.rb

```diff
@@ -59,6 +59,23 @@ def self.supported
            :java_start_exception),
       rule(:java_after_exception, /^[\r\n]*$/, :java_after_exception),
       rule([:java_after_exception, :java], /^[\t ]+(?:eval )?at /, :java),
+
+      # C# nested exception
+      rule([:java_after_exception, :java],
+           /^[\t ]+--- End of inner exception stack trace ---/,
+           :java),
+
+      # C# aggregate exception
+      rule([:java_after_exception, :java],
+           /^---> \(Inner Exception #[0-9]+\)/,
+           :java),
+
+      # C# exception from async code
+      rule([:java_after_exception, :java],
+           %r{^---\sEnd\sof\sstack\strace\sfrom\sprevious\s
+           location\swhere\sexception\swas\sthrown\s---}x,
```

Comment:
The linter doesn't like this ('Use // around regular expression.'), but without it I end up with a line that is too long for the linter. Have you got any suggestions about how to split the regex across multiple lines (Ruby isn't my language)?

I could perhaps define a '+' operator for Regexp, as suggested [here](https://www.ruby-forum.com/t/concatenation-of-regular-expressions/87885/4).

Reply:
Try the `/x` modifier, as suggested [here](https://stackoverflow.com/questions/3762183/split-ruby-regex-over-multiple-lines).

Reply:
Thanks. Done.